index.html

```diff
@@ -53,6 +53,315 @@ <h1>I'm Christian <br>
+      <p>About Myself</p>
+    </div>
+    <div class="self-description">
+      <p>Hello I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    </div>
+    <div class="connect">
+      <p>LET'S CONNECT</p>
+    </div>
+    <div class="media-icons">
+      <ul class="icons">
+        <li>
+          <a href=""><img src="./images/media_icons/twitter.svg" alt="twitter icon"></a>
+        </li>
+        <li>
+          <a href=""><img src="./images/media_icons/linkedin.svg" alt="linkedin icon"></a>
+        </li>
+        <li>
+          <a href=""><img src="./images/media_icons/mail.svg" alt="mail icon"></a>
+        </li>
+        <li>
+          <a href=""><img src="./images/media_icons/github.svg" alt="github icon"></a>
+        </li>
+        <li>
+          <a href=""><img src="./images/media_icons/hand.svg" alt="other icon"></a>
+        </li>
+      </ul>
+    </div>
+    <div class="about-myself-button">
+      <a href="">
+        <button class="see-project-button">See Project</button>
+      </a>
+    </div>
+    <div class="dropdown-menu">
+      <ul class="list1">
+        <li class="language-name">
+          <div class="positioning">
+            <div>
+              <a href="">Languages</a>
+            </div>
+          </div>
+        </li>
+            <ul class="languages-dropdown">
+              <li>
+                <div class="html-css-js">
+                  <div class="language-icons">
+                    <img src="./images/Languages/js.svg" alt="">
+                  </div>
+                  <div class="language-tags">                  
+                    <p>JavaScript</p>
+                  </div>
+                </div>
+              </li>
+              <li>
+                <div class="html-css-js">
+                  <div class="language-icons">
+                    <img src="./images/Languages/html.svg" alt="">
+                  </div>
+                  <div class="language-tags">                  
+                    <p>CSS</p>
+                  </div>
+                </div>
+              </li>
+              <li>
+                <div class="html-css-js">
+                  <div class="language-icons">
+                    <img src="./images/Languages/css.svg" alt="">
+                  </div>
+                  <div class="language-tags">                  
+                    <p>HTML</p>
+                  </div>
+                </div>
+              </li>
+            </ul>
+        <li class="language-name">
+          <div class="positioning">
+            <div>
+              <a href="">Frameworks</a>
+            </div>
+          </div>
+        </li>
+        <li class="language-name">
+          <div class="positioning">
+            <div>
+              <a href="">Skills</a>
+            </div>
+          </div>
```

Comment:
- Hey, please, kindly add the missing dropDown image close to the framework and skills to match the design as much as you can below:

*Figma design*
![Screenshot from 2023-01-14 01-43-05](https://user-images.githubusercontent.com/59397159/212442108-bf10b1ce-47a6-4063-bc0f-4ceb7c56fe8b.png)

*Your design*
![Screenshot from 2023-01-14 01-43-32](https://user-images.githubusercontent.com/59397159/212442106-1fea16d6-5906-40ec-9f74-0ff784bb1d1c.png)

Reply:
@zieeco Thank you so much for the feedback 👍🏼
I made the changes and I am ready to request a review.